frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -484,8 +484,7 @@ func toDispatchState(ctx context.Context, dt []byte, opt ConvertOpt) (*dispatchS
 								d.dispatched = true
 								d.state = *st
 								if img != nil {
-									// timestamps are inherited as-is, regardless to SOURCE_DATE_EPOCH
-									// https://github.com/moby/buildkit/issues/4614
+									img.Created = nil
```

Comment:
@AkihiroSuda This comment was from https://github.com/moby/buildkit/pull/4663 . Iiuc created time needs to be only maintained for `d.baseImg` .